packages/react-i18n/src/i18n.ts

```diff
@@ -582,7 +582,7 @@ export class I18n {
       hour: 'numeric',
     });
 
-    const zoneMatchGroup = /\s(\w*$)/i.exec(hourZone);
+    const zoneMatchGroup = /\s([\w()+-:]*$)/i.exec(hourZone);
```

Comment:
[GMT+9.5](http://www.timebie.com/timelocal/gmt+95.php) uses a decimal point

Reply:
(We may need to include `.` in the set)

Reply:
Also, we should probably use `+` instead of `*`. The result will be the same: there will be a match but the matched group will yield a `''` if there's nothing following the `\s`. But using a `+` would make this more intuitive.

Reply:
It still matched `1.5` i guess since its still a number but I added into the set just in case :)

Reply:
Im gonna have to figure out how to not make these a range of chars

Reply:
i made it `/\s([\w(+\-:.)]+$)/